package.json

```diff
@@ -28,7 +28,7 @@
         "@typescript-eslint/eslint-plugin": "^6.15.0",
         "@typescript-eslint/parser": "^6.15.0",
         "aws-cdk": "2.110.1",
```

Comment:
I think it's worth updating "aws-cdk" to the same version if possible

Reply:
done